docs/fabric-native-components-android.md

```diff
@@ -0,0 +1,391 @@
+import com.facebook.react.uimanager.SimpleViewManager
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.ViewManagerDelegate
+import com.facebook.react.uimanager.annotations.ReactProp
+import com.facebook.react.viewmanagers.RTNCenteredTextManagerInterface
+import com.facebook.react.viewmanagers.RTNCenteredTextManagerDelegate
+
+@ReactModule(name = CenteredTextManager.REACT_CLASS)
+class CenteredTextManager(context: ReactApplicationContext) : SimpleViewManager<CenteredText>(), RTNCenteredTextManagerInterface<CenteredText> {
+  private val delegate: RTNCenteredTextManagerDelegate<CenteredText, CenteredTextManager> =
+    RTNCenteredTextManagerDelegate(this)
+
+  override fun getDelegate(): ViewManagerDelegate<CenteredText> = delegate
+
+  override fun getName(): String = REACT_CLASS
+
+  override fun createViewInstance(context: ThemedReactContext): CenteredText = CenteredText(context)
+
+  @ReactProp(name = "text")
+  override fun setText(view: CenteredText, text: String?) {
+    view.text = text
+  }
+
+  companion object {
+    const val REACT_CLASS = "RTNCenteredText"
+  }
+}
+
+```
+
+</TabItem>
+</Tabs>
+
+#### Add: RTNCenteredTextPackage
+
+<Tabs groupId="android-language" queryString defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="java">
+
+```java title="Demo/android/src/main/java/com/rtncenteredtext/CenteredTextPackage.java"
+package com.rtncenteredtext;
+
+import com.facebook.react.TurboReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.module.model.ReactModuleInfo;
+import com.facebook.react.module.model.ReactModuleInfoProvider;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CenteredTextPackage extends TurboReactPackage {
+  @Override
+  public List<ViewManager<?, ?>> createViewManagers(ReactApplicationContext reactContext) {
+    return Collections.singletonList(new CenteredTextManager(reactContext));
+  }
+
+  @Override
+  public NativeModule getModule(String s, ReactApplicationContext reactApplicationContext) {
+    if (CenteredTextManager.REACT_CLASS.equals(s)) {
+      return new CenteredTextManager(reactApplicationContext);
+    }
+    return null;
+  }
+
+  @Override
+  public ReactModuleInfoProvider getReactModuleInfoProvider() {
+    return new ReactModuleInfoProvider() {
+      @Override
+      public Map<String, ReactModuleInfo> get() {
+        Map<String, ReactModuleInfo> map = new HashMap<>();
+        map.put(CenteredTextManager.REACT_CLASS, new ReactModuleInfo(
+                CenteredTextManager.REACT_CLASS,
+                CenteredTextManager.REACT_CLASS,
+                false,
+                false,
+                false,
+                BuildConfig.IS_NEW_ARCHITECTURE_ENABLED
```

Comment:
perhaps we should add a comment with what these parameters are: why do we have two REACT_CLASS? what are the three `false`s?